src/wallet/wallet.h

```diff
@@ -962,11 +976,23 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      */
     static CAmount GetMinimumFee(unsigned int nTxBytes, unsigned int nConfirmTarget, const CTxMemPool& pool, const CBlockPolicyEstimator& estimator, FeeCalculation *feeCalc = nullptr, bool ignoreGlobalPayTxFee = false);
     /**
+     * Estimate the minimum fee rate considering user set parameters
+     * and the required fee
```

Comment:
perhaps note it doesn't have the maxtxfee check inside it, making it slightly asymmetrical to the total fee one.